WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeCreateCampaignIntroView.swift

```diff
@@ -0,0 +1,233 @@
+                            .largeTitleStyle()
+                            .multilineTextAlignment(.center)
+
+                        Text(Localization.subtitle)
+                            .foregroundColor(Color(.secondaryLabel))
+                            .bodyStyle()
+                            .multilineTextAlignment(.center)
+
+                        Image(uiImage: UIImage.blazeIntroIllustration)
+                    }
+
+                    VStack(spacing: Layout.featuresVerticalSpacing) {
+                        ForEach(features, id: \.title) { feature in
+                            FeatureView(feature: feature)
+                        }
+                    }
+                }
+                .padding(Layout.contentPadding)
+            }
+            .safeAreaInset(edge: .bottom) {
+                VStack(spacing: Layout.CTAStack.spacing) {
+                    Divider()
+                        .frame(height: Layout.CTAStack.dividerHeight)
+                        .foregroundColor(Color(.separator))
+
+                    Button(Localization.createYourCampaign) {
+                        onCreateCampaign()
+                    }
+                    .buttonStyle(PrimaryButtonStyle())
+                    .padding(.horizontal, Layout.CTAStack.buttonHPadding)
+
+                    Button(Localization.learnHowBlazeWorks) {
+                        // TODO: 11566
+                    }
+                    .buttonStyle(LinkButtonStyle())
+                    .padding(.horizontal, Layout.CTAStack.buttonHPadding)
+                }
+                .background(Color(UIColor.systemBackground))
+            }
+        }
+    }
+}
+
+private extension BlazeCreateCampaignIntroView {
+    struct Feature {
+        let title: String
+        let subtile: String
+    }
+
+    struct FeatureView: View {
+        /// Scale of the view based on accessibility changes
+        @ScaledMetric private var scale: CGFloat = 1.0
+
+        let feature: Feature
+
+        var body: some View {
+            HStack {
+                VStack(alignment: .leading) {
+                    Image(uiImage: .blaze)
+                        .resizable()
+                        .renderingMode(.template)
+                        .foregroundColor(Color(.brand))
+                        .frame(width: Layout.logoSize * scale, height: Layout.logoSize * scale)
+
+                    Text(feature.title)
+                        .fontWeight(.bold)
+                        .bodyStyle()
+
+                    Text(feature.subtile)
+                        .bodyStyle()
+                }
+
+                Spacer()
+            }
+        }
+    }
+}
+
+private extension BlazeCreateCampaignIntroView {
+    enum Localization {
+        static let headline = NSLocalizedString(
+            "blazeCreateCampaignIntroView.headline",
+            value: "Blaze",
+            comment: "Headline text for the Blaze campaign intro view"
+        )
```

Comment:
I don't think Blaze should be localized.

Reply:
Good point. Done in e3fba1ff1885167eea289586f536e3e87236ea27